Content.Server/Anomaly/Effects/TechAnomalySystem.cs

```diff
@@ -37,7 +37,7 @@ public override void Update(float frameTime)
             if (_timing.CurTime < tech.NextTimer)
                 continue;
 
-            tech.NextTimer += TimeSpan.FromSeconds(tech.TimerFrequency);
+            tech.NextTimer = _timing.CurTime + TimeSpan.FromSeconds(tech.TimerFrequency);
```

Comment:
I'm pretty sure sloth always complains in reviews when an update loop is done like this because it eats the remainder.
@metalgearsloth  could you elaborate on this?

Reply:
Yeah, the standard way to do it is to add the frequency because otherwise different tickrates give different outcomes, e.g. if you expect 60 ticks a minute but suddenly start getting 59.